src/bindings/python/src/openvino/runtime/ie_api.py

```diff
@@ -262,7 +262,7 @@ def create_infer_request(self) -> InferRequest:
         """
         return InferRequest(super().create_infer_request())
 
-    def infer_new_request(self, inputs: Union[dict, list, tuple, Tensor, np.ndarray] = None) -> OVDict:
+    def infer_new_request(self, inputs: Any = None) -> OVDict:
```

Comment:
Should we have `Any`? It doesn't say much, and now there's a discrepancy between Python typing and the docstring.

Reply:
I will change a docstring then. It is the same as for all other infer functions -- just aligned them.

Reply:
Done.